data/announcements/scheduled.yaml

```diff
@@ -164,7 +164,21 @@ announcements:
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     April 18 - April 21, 2023.
-    
+
+- name: Kubecon 2023 NA
+  startTime: 2023-10-20T00:00:00  #Added in https://github.com/kubernetes/website/pull/43565
+  endTime: 2023-11-09T18:00:00
+  style: >-
+    background: linear-gradient(90deg, rgba(55,38,93,1) 0%, rgba(85,53,125,1) 65%, rgba(33,70,139,1) 100%);
+    color: #fffff;
+  title: |
+    <img src ="/images/announcements/kccnc-na-2023-white.svg" style="float: right; height: 80px;" />
+    <a href= "https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/">KubeCon + CloudNativeCon NA 2023</a> <em> Chicago, Illinois + Virtual</em>.
+  message: |
+    4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
+    November 6 - November 9, 2023.
+
+
```

Comment:
_nit:_ Could we possibly move this new announcement block to the end of the file? This way, the announcements in the file would be in some _(chronological)_ order for better readability, even though I know that we have added some older announcements in between out of order to accommodate scheduling preferences for overlapping dates.

Reply:
Good call, updated.